lib/dialects/mysql/connection-manager.js

```diff
@@ -82,6 +81,11 @@ class ConnectionManager extends AbstractConnectionManager {
       supportBigNumbers: true
     };
 
+    // -FOUND_ROWS used by default but can be suppressed
+    if (!config.dialectOptions || config.dialectOptions.foundRows !== false) {
```

Comment:
@janmeier @sushantdhiman Apologies for the delay. I switched to `dialectOptions.flags` which effectively means this PR only adds a test. This is now ready for review again.